stock_picking_invoicing/README.rst

```diff
@@ -0,0 +1,42 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
I did because of travis errors. Module was/is requirement for stock_picking_invoicing_incoterm. Module exists in 7.0 and in 9.0 (although not installable). In 8.0 the module was there with i18n files. Now I understand the proper fix is to remove the requirement from stock_picking_invoicing_incoterm.